modules/ROOT/pages/transaction-management.adoc

```diff
@@ -146,6 +145,103 @@ When running in a transactional scope, the entire process must run in the same t
 
 Any other router or scope (for example, Foreach) continues with the transaction and its behavior is not modified.
 
+== Error Handling
+
+When working with transactions, we need to know how to handle errors. This includes to continue with the transaction even if an specific error occurs, or to handle correctly the rollback. 
```

Comment:
Something about this wording bothers me (the `we`, for instance). Can you check, @Cristian-Venticinque ?

Reply:
Rewrote this in PR: #621:
"When an error occurs during a transaction, your application must be able to handle the error and continue with the transaction or do a rollback."